pkg/config/generate.go

```diff
@@ -584,3 +584,18 @@ func randomString(length int) (string, error) {
 
 	return string(b), nil
 }
+
+func selectDNSNames(cs *acsapi.ContainerService, c *Config) {
```

Comment:
Config is no more. Also, it would be nice to start seeing unit tests about these functions. We can start with this one.